src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsEmbeddedFractures.hpp

```diff
@@ -111,23 +128,29 @@ class SolidMechanicsEmbeddedFractures : public SolverBase
 
     constexpr static auto fractureRegionNameString = "fractureRegionName";
 
+    constexpr static auto fractureTractionString = "fractureTraction";
+
+    constexpr static auto dTraction_dJumpString = "dTraction_dJump";
+
   } SolidMechanicsEmbeddedFracturesViewKeys;
 
+  string & getContactRelationName() { return m_contactRelationName; };
+
+  string const & getContactRelationName() const { return m_contactRelationName; };
+
+  string & getFractureRegionName() { return m_fractureRegionName; };
```

Comment:
do you need a non-const accessor?

Reply:
No, I don't use it. I have only added it for completeness.